packages/bundler/src/Bundler.ts

```diff
@@ -21,19 +24,40 @@ export class Bundler implements IBundler {
      * @returns Promise<UserOpGasPricesResponse>
      */
     async getUserOpGasFields(userOp: UserOperation, chainId: ChainId): Promise<UserOpGasFieldsResponse> {
-        // transformUserOP will convert all bigNumber values to string
-        const userOperation = transformUserOP(userOp)
-        const response: any = await sendRequest({
-            url: `${this.bundlerConfig.bundlerUrl}`,
+        // TODO: will be removed once full userOp requirement is removed from bundler side
+        const dummpyUserop = {
+            callGasLimit: '0',
+            verificationGasLimit: '0',
+            preVerificationGas: '0',
+            maxFeePerGas: '0',
+            maxPriorityFeePerGas: '0',
+            paymasterAndData: '0x',
+            signature: '0x'
+        }
+        const userOperation = { ...dummpyUserop, ...userOp }
+        userOp = transformUserOP(userOperation)
+        console.log('userOp sending for fee estimate ', userOp);
+
+        const response: GetUserOpGasFieldsResponse = await sendRequest({
+            url: `${this.bundlerConfig.bundlerUrl}/${chainId}/${this.bundlerConfig.apiKey}`,
             method: HttpMethod.Post,
             body: {
-                method: 'eth_getUserOpGasFields',
-                params: [userOperation, this.bundlerConfig.epAddress, chainId],
-                id: 1234,
+                method: 'eth_estimateUserOperationGas',
+                params: [userOp, this.bundlerConfig.epAddress],
+                id: getTimestampInSeconds(),
                 jsonrpc: '2.0'
             }
         })
-        return response
+
+        const userOpGasFieldsResponse = response.result
+        for (const key in userOpGasFieldsResponse) {
+            if (key === 'maxFeePerGas' || key === 'maxPriorityFeePerGas' || key === 'gasPrice')
```

Comment:
why gasPrice is still coming into picture. In case of non1559 networks, bundler should ensure to send same values of maxFeePerGas and maxPriorityFeePerGas

Reply:
yeah. its removed from bundler response but left in this check so have removed from here as well